src/pages/testResult.tsx

```diff
@@ -1,7 +1,190 @@
+import styled from '@emotion/styled';
 import type { NextPage } from 'next';
+import { useState, useEffect } from 'react';
+import Loading from '@/components/Common/Loading';
+import Button from '@/components/Input/Button';
+import { initMSW } from '@/lib/msw';
 
-const testResult: NextPage = () => {
-  return <div>결과화면입니다 !!</div>;
+initMSW();
+
+type InfoType = {
+  img: string;
+  title: string;
+  content: string;
+};
+
+const levelInfo: { [key: number]: InfoType } = {
+  1: {
+    img: '',
+    title: '당신의 불타는 똥꼬',
+    content:
+      '다음 날 아침 화장실에서 심판을 받는다. 심판은 고통스럽고 따갑고 뜨겁다. 매콤함이 항문으로도 느껴지는 것만 같다..',
+  },
+  2: {
+    img: '',
+    title: '빠빠빨간맛..궁금해 허니',
+    content:
+      '지금 흐르는건 땀일까 콧물일까  어질한 매운맛에 정신 못차리다가도  어느새 다시 찾는 나.. 조금 빠져버린걸지도?',
+  },
+  3: {
+    img: '',
+    title: '군침이 싹 도네',
+    content: '빨간색만 보면 군침이 싹도네  그만큼 맛있다는거지..',
+  },
+  4: {
+    img: '',
+    title: '한국인이면 매운맛이지',
+    content:
+      '가능한..빨간색이 보이게..!  이제 색깔이 빨갛지 않은 요리는 맛없게 느껴진다,,',
+  },
+  5: {
+    img: '',
+    title: 'K-입맛 그자체',
+    content: "매운 맛을 찢어버린 캡사이'신'  당신이 K-입맛입니다.",
+  },
+};
+
+const TestResult: NextPage = () => {
+  const [isResult, setIsResult] = useState<boolean>(false);
```

Comment:
```suggestion
  const [isResult, setIsResult] = useState(false);
```
요 부분 어차피 타입 추론되서 타입 생략해도 될 것 같아용 ㅎㅎ